### DISCUSSION TITLE
Check drag on Flare

### DISCUSSION THREAD

**Discussion by mga010** [flybywiresim/aircraft#2439] [Help] [answered]
It is quite difficult to prove. But I think that the plane should bleed off airspeed (and thus lose lift) much faster on landing. The flare feels eternal.

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
The flight model has been designed from performance charts. Obviously there are a few hiccups but in the future they'll be ironed out for sure.

**Reply by mga010** [reply]
My experience with the add-on so far says that you are right. It was completely off at the start and now it is quite accurate.

**Comment by hotshotp** [comment]
this should feel much better when you use the fly by wire system branch, it has eliminated any of the annoying floaty issues and feels much better
